easybuild/easyconfigs/g/Ghostscript/Ghostscript-9.14-intel-2014b.eb

```diff
@@ -10,9 +10,11 @@ description = """Ghostscript is a versatile processor for PostScript data with t
 toolchain = {'name': 'intel', 'version': '2014b'}
 toolchainopts = {'pic': True}
 
-source_urls = ["http://downloads.ghostscript.com/public/"]
+source_urls = ["http://downloads.ghostscript.com/public/old-gs-releases/"]
 sources = [SOURCELOWER_TAR_GZ]
 
+checksums = ['024f040b1371348a2b7fc4d25b379d33']
```

Comment:
this looks wrong, I'm getting `586494befb443363338c1b6379f13973` for this download...

Both when I try to redownload it, and with the download I had before:

```
$ ls -l ghostscript-9.14.tar.gz 
-rw-rw-r-- 1 myuser myuser 35514406 Dec 10  2014 ghostscript-9.14.tar.gz
$ md5sum ghostscript-9.14.tar.gz
586494befb443363338c1b6379f13973  ghostscript-9.14.tar.gz

```